llama_index/callbacks/token_counting.py

```diff
@@ -48,6 +48,9 @@ def get_llm_token_counts(
         try:
             usage = response.raw["usage"]  # type: ignore
 
+            if usage is None:
+                raise ValueError("Invalid usage!")
```

Comment:
We don't want to raise an error in the callback -- if usage is None, we should count the `response` in the message no?

Reply:
I oriented myself on the handling of the case that messages_tokens == 0. However I agree on the premise whether raising an ValueError is the correct way of handling it. There is just currently an issue that if usage is None that the access on prompt_tokens fails

Reply:
OK I now set default values to message and response_tokens and update their value only if usage is not None. This should be cleaner

Reply:
Thanks!